frontend/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "table-soccer-tournaments-frontend",
   "version": "0.1.0",
   "description": "Table Soccer Tournaments React Web App",
+  "homepage": "/CodersCamp2020.Project.FullStack-Node-React.TableSoccerTournaments",
```

Comment:
Hmmm, a to ciekawe. Chyba powinno byc jakos konfigurowane zaleznie od srodowiska? Lokalnie teraz sie nic nie popsulo?

Reply:
Wyglada na to jakby wystarczylo zostawic "."

https://stackoverflow.com/questions/43011207/using-homepage-in-package-json-without-messing-up-paths-for-localhost

Reply:
O widzisz tego nie doczytałem :) i najwidoczniej z tym tokenem również, założyłem, że to my sobie ustawiamy jakiś token do deployu na gh pages :p za chwilę to nareperuję!